docs/reference/secure-connection.md

```diff
@@ -292,13 +291,22 @@ Tips for creating API keys:
 * {{ls}} can send both collected data and monitoring information to {{es}}. If you are sending both to the same cluster, you can use the same API key. For different clusters, you need an API key per cluster.
 * A single cluster can share a key for ingestion and monitoring purposes.
 * A production cluster and a monitoring cluster require separate keys.
+* When you create an API key for {{ls}}, select **Logstash** from the **API key format** dropdown.
+  This option formats the API key in the correct `id:api_key` format required by {{ls}}.
+
+  :::{image} images/logstash_api_key_format.png
+  :alt: API key format dropdown set to {{ls}}:
+  :screenshot:
+  :width: 400px
+  :::
```

Comment:
@jsvd, is this statement accurate for self-managed as well as serverless and hosted?

Reply:
I believe that line 303: "The UI for API keys may look different depending on the deployment type" covers us for Serverless and Hosted.  What about self-managed? Does this apply? 

If so, we can leave the statement as is with the line 303 caveat. 
If not. we can specify Serverless and Hosted at the beginning of this bullet item.

Reply:
In self-hosted, while the interface to arrive at API creation and for API creation itself may be different, the way to pick the Logstash formatted version is the same:

<img width="852" alt="Screenshot 2025-06-27 at 15 33 35" src="https://github.com/user-attachments/assets/48982cf0-97ff-4d59-8a15-f28d1531a91e" />

Reply:
@jsvd, thanks for clarifying! Much appreciated.